packages/special-pages/index.mjs

```diff
@@ -56,6 +56,11 @@ export const support = {
         'integration': ['copy', 'build-js'],
         'apple': ['copy', 'build-js', 'inline-html'],
     },
+    /** @type {Partial<Record<ImportMeta['injectName'], string[]>>} */
+    'new-tab': {
+        'integration': ['copy', 'build-js'],
+        'windows': ['copy', 'build-js'],
+    },
 }
```

Comment:
We can include this on windows early - it won't be used by default but helps them in their integration